### ISSUE TITLE
Bees don't spawn at all

### ISSUE THREAD

**Issue by kittykhajiit** [JDKDigital/productive-bees#249]
Hi I have an ATM7 server and I can't find bees ANYWHERE? I've checked hundreds of chunks and not a single bee. How do I know if the mod is spawning them in properly?

**Comment by Nekosity12** [comment]
What kind of bees we talking here? Vanilla? If so the mod doesn't handle those. Wild? There's only slimy/ender and a few nether wild bees so they'd be pretty rare unless looking in the right areas. Solitary? These are usually best spawned in yourself with a nest (Don't be fooled by "spawned in" and thinking creative/commands, you can spawn them in using the mod itself) Just take a nest, see what biomes it can be in and what bees use it by using JEI then feed it a honey treat. 

Obviously you still need a bee first to produce honey before you can do this however. Fear not for theres a way to spawn in vanilla bees too which is actually purely vanilla! Simply have an oak or birch sapling near a couple flowers (I believe within 2 blocks) and then bonemeal the saplings. It's only like a 5% chance or smtg so may take multiple tries but you'll eventually get vanilla bees! You can additionally craft a Nest Locator (looks like a music box compass) to help you find nests that bees may have occupied. In order to use it, shift right click (with the nest locator) a block that resembles the type of nest you're looking for (So grass block for grass nest or oak log for oak log nest) and then right click while holding the nest locator and it will tell you if there are any nests nearby. The compass part of the item will point in the direction you need to walk and chat will tell you how far!

**Comment by kittykhajiit** [comment]
I've played this mod before on an old ATM server and bees would be everywhere (like mason bees, zombee, rgbee etc) but I can't find a single one. I can't see any hives or anything either. I loaded into a singleplayer world and found a bee almost instantly. I'm wondering if they aren't spawning in on my server for some reason?

edit: think I'm thinking of resourceful bees rather than productive bees.. but I still can't find any haha
